response_operations_ui/views/reporting_units.py

```diff
@@ -60,7 +60,11 @@ def view_reporting_unit(ru_ref):
         info_message = f'Response status for {survey["surveyRef"]} {survey["shortName"]}' \
                        f' period {period_arg} changed to {new_status}'
 
-    return render_template('reporting-unit.html', ru=ru_details['reporting_unit'], surveys=ru_details['surveys'],
+    if request.args.get('info'):
+        info_message = 'Verification email re-sent'
+
+    return render_template('reporting-unit.html', ru_ref=ru_ref, party_id=respondent['id'],
```

Comment:
`party_id` assumes `respondent` will be assigned by the for-loops above. Is there an alternative to avoid a 500 (UnboundLocalError)?